app/services/ccms/attribute_value_generator.rb

```diff
@@ -121,7 +121,7 @@ def child_subject_of_proceeding?(_options)
       # as Client Involvement Type or using the merits task, just that there is a child subject on the application
       legal_aid_application.proceedings.any? do |proceeding|
         proceeding.special_childrens_act? &&
-          (proceeding.relationship_to_child.eql?("child_subject") || proceeding.client_involvement_type_ccms_code.eql?("W"))
+          (legal_aid_application.applicant.relationship_to_children.eql?("child_subject") || proceeding.client_involvement_type_ccms_code.eql?("W"))
```

Comment:
just wondering, from a business logic point of view whether we need to check that proceeeding is `special_childrens_act?`. 🤔 If proceeding has a child subject `client_involvement_type_ccms_code`, or [now] the application has a child subject `relationship_to_children` does that qualify it regardless of matter type :shrug:


If so could refactor, but either way. The name of the method also seems less aligned now. `child_subject_of_application?` or just `child_subject?`

```ruby
 def child_subject?(_options)
   return true if legal_aid_application.applicant.relationship_to_children.eql?("child_subject")
   
   legal_aid_application.proceedings.any? do |proceeding|
     proceeding.relationship_to_child.eql?("child_subject")
   end
 end
```

Reply:
I can see the point of the logic update and will investigate

With regards to renaming, this method is only used to populate a CCMS field named `CLIENT_CHILD_SUBJECT_OF_PROC` so I think it makes sense to leave it as is for now :+1: